base/expr.jl

```diff
@@ -922,6 +922,26 @@ function remove_linenums!(src::CodeInfo)
     return src
 end
 
+replace_linenums!(ex, ln::LineNumberNode) = ex
+function replace_linenums!(ex::Expr, ln::LineNumberNode)
+    if ex.head === :block || ex.head === :quote
+        # replace line number expressions from metadata (not argument literal or inert) position
+        map!(ex.args, ex.args) do x
```

Comment:
```suggestion
        map!(ex.args, ex.args) do @nospecialize(x)
```

Reply:
Out of curiosity, why don't we do that above in `remove_linenums!`?

Reply:
someone forgot